cove_360/templates/cove_360/explore.html

```diff
@@ -184,7 +184,9 @@ <h3 class="panel-title panel-title-explore" id="conversion-title">
             <p class="explanation">{% trans "Before checking your data we needed to convert it to JSON." %}</p>
             <div class="explore-help">This is because 360Giving is an open data standard which uses a <a href="http://json-schema.org/">JSON Schema</a> to describe the standard in a technical way. 
 		    <br/>
-		    If a file cannot be converted to JSON it indicates that it cannot be correctly mapped to the standard and needs to be reviewed. If this is the case you should check the file and re-upload it once you’ve fixed the problem. The conversion errors below will give an indication of where the issue is. 
+		    {% if conversion_error %}
```

Comment:
Note that the interface may say "This data could not be read as 360Giving JSON data" because `conversion_warning_messages` is truthy, but this message will not be shown. However, we can read it as JSON (there's just problems with that conversion), so maybe its the other message, and not this one, that's the problem.